src/Illuminate/View/Compilers/Compiler.php

```diff
@@ -26,9 +27,15 @@
      * @param  \Illuminate\Filesystem\Filesystem  $files
      * @param  string  $cachePath
      * @return void
+     *
+     * @throws \InvalidArgumentException
      */
     public function __construct(Filesystem $files, $cachePath)
     {
+        if (! $cachePath) {
+            throw new InvalidArgumentException('The cache path must be non-empty.');
```

Comment:
Indeed, one of these may be better:

* `The cache path must be a non-empty value.`
* `The cache path must not be an empty value.`